docs/source/stability_policy.rst

```diff
@@ -0,0 +1,173 @@
+Stability Policy
+================
+
+.. important::
+
+    This stability policy is in place since version NEXT.VERSION.
+    While earlier versions of ``python-telegram-bot`` also had stable interfaces, they had no explicit stability policy and hence did not follow the rules outlined below in all detail.
+    Please also refer to the :ref:`changelog <ptb-changelog>`.
+
+.. caution::
+
+    Large parts of the :mod:`telegram` package are the Python representations of the Telegram Bot API, whose stability policy PTB can not influence.
```

Comment:
```suggestion
    Large parts of the :mod:`telegram` package are the Python representations of the Telegram Bot API, which stability policy PTB can not influence.
```

Reply:
Didn't we go over this before? 😄

Reply:
https://github.com/python-telegram-bot/python-telegram-bot/pull/3622#discussion_r1161248368